pyproject.toml

```diff
@@ -51,6 +51,7 @@ dependencies = [
     "scipy>=1,<2",
     "numpy>=1.17.0",
     "filelock>=3.15",
+    "libgcc<15 ; platform_system == 'Windows'",
```

Comment:
The c backend is optional for PyTensor, this is something you want to do at the conda level not here

Reply:
Ya, libgcc isn't even on PyPI. Would anyone be available for a quick call to explain to me what's going on?

Reply:
Yep I quickly discovered this PR is extremely stupid :)

I can close it once we have another one addressing the actual issue.